reference.md

```diff
@@ -120,6 +112,14 @@ operating system
 :   Software that manages interactions between users, hardware, and software [processes](#process).
     Common examples are Linux, macOS, and Windows.
 
+option
+:   A way to specify an argument or setting to a command-line program.
+    By convention Unix applications use a dash followed by a single letter,
+    such as `-v`, or two dashes followed by a word, such as `--verbose`,
+    while DOS applications use a slash, such as `/V`.
```

Comment:
I'm not sure we need to go into DOS in the Unix shell lesson.

Reply:
I more or less just moved the existing text; that change is, IMO, orthogonal to this diff.

Reply:
Yes, fair point!

Reply:
As a follow-up, this may be a good opportunity to flag something for "good first commit" status for purposes of onboarding.